.github/workflows/test.yml

```diff
@@ -39,19 +44,25 @@ jobs:
           name: replay-driver
           path: macOS-recordreplay.so
   preview-branch:
-    name: Wait for Vercel Preview Branch
+    name: Vercel Preview Build
     runs-on: ubuntu-latest
     steps:
-      - name: Waiting for 200 from the Vercel Preview
-        uses: patrickedqvist/wait-for-vercel-preview@v1.1.1
-        id: waitFor200
+      - uses: actions/checkout@v2
+      - uses: amondnet/vercel-action@v20
+        id: vercel-deploy
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          max_timeout: 500
-          check_interval: 30
-      - run: echo ${{steps.waitFor200.outputs.url}}
+          # for posting a comment upon completion
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          vercel-token: "aLNsahI9JClTxsQuxV7Kf5SD"
+          vercel-org-id: "team_o3Dz7BlGrXUzHeSUxDXdLfxP"
+          vercel-project-id: "prj_t1WZoKI1cEWvD9FAfXw7JXuspF7y"
```

Comment:
Honestly it would be fine by me to just shove these into secrets too. There's not really any reason to broadcast an ID that could be used to target our services if we don't have to.